crates/codegen/parser_runtime/src/kinds.rs

```diff
@@ -0,0 +1,54 @@
+#[cfg(feature = "slang_napi_interfaces")]
+use {napi::bindgen_prelude::*, napi_derive::napi};
+
+#[derive(
+    Debug,
+    Eq,
+    Ord,
+    PartialEq,
+    PartialOrd,
+    serde::Serialize,
+    strum_macros::AsRefStr,
+    strum_macros::Display,
+    strum_macros::EnumString,
+)]
+#[cfg_attr( feature = "slang_napi_interfaces", /* derives `Clone` and `Copy` */ napi(string_enum, namespace = "legacy") )]
```

Comment:
should we also remove `namespace = "legacy"` here if they are removed from the rest?

Reply:
These BTW are never used. This is a dummy file in this crate that is never copied.

Reply:
Not a blocker, but I think we should keep it up to date with the generated sources, to eliminate confusion.